docs/src/AlgebraicGeometry/intro.md

```diff
@@ -23,6 +23,12 @@ General textbooks offering details on the theory of varieties and schemes includ
 - [The Stacks Project](https://stacks.math.columbia.edu)
 
 
+## Tutorials
+
+We encourage you to take a look at our [tutorials](https://www.oscar-system.org/tutorials/). For instance,
+there is a [tutorial for toric geometry in OSCAR](https://nbviewer.org/github/oscar-system/oscar-website/blob/gh-pages/tutorials/ToricGeometryInOSCAR.ipynb).
```

Comment:
Why not also this?
```suggestion
there is a [tutorial for toric geometry in OSCAR](https://www.oscar-system.org/tutorials/ToricGeometry/).
```

Reply:
Slipped my attention. Thx for the pointer!